include/highfive/bits/H5Converter_misc.hpp

```diff
@@ -392,39 +362,53 @@ struct data_converter<FixedLenStringArray<N>, void>
     using container_converter<FixedLenStringArray<N>, char>::container_converter;
 };
 
+template <typename T>
+struct storage {
```

Comment:
I don't quite understand what is the purpose of this class, do we really need it? Also, if it's really only needed for the `data_converter<std::vector<Reference.>`, why not make it an inner class?

Reply:
Little by little it will be used by all classes. And data_converter will be deleted.